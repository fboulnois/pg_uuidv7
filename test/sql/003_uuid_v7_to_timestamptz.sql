-- set up variables for testing
\set uuidv7 '018570bb-4a7d-7000-8000-000000000000'
\set uuidtz '2023-01-02 04:26:40.637+00'
\set uuid '2023-01-02 04:26:40.637'

-- test converting a uuid to a timestamptz
SELECT uuid_v7_to_timestamptz(:'uuidv7') = :'uuidtz';

-- test converting a uuid to a timestamp
SELECT uuid_v7_to_timestamp(:'uuidv7') = :'uuid';
