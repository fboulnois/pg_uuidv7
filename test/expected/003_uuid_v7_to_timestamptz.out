-- set up variables for testing
\set uuidv7 '018570bb-4a7d-7000-8000-000000000000'
\set uuidtz '2023-01-02 04:26:40.637+00'
-- test converting a uuid to a timestamp
SELECT uuid_v7_to_timestamptz(:'uuidv7') = :'uuidtz';
 ?column? 
----------
 t
(1 row)

